include/functions.h

```diff
@@ -836,20 +842,21 @@ void func_800B8C50(Actor* actor, GlobalContext* globalCtx);
 void func_800B8D98(GlobalContext* globalCtx, Actor* actor, f32 param_3, s16 param_4, f32 param_5);
 // void func_800B8DD4(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5, UNK_TYPE4 param_6);
 // void func_800B8E1C(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5);
-// void func_800B8E58(void);
+void func_800B8E58(Actor* actor, s16 sfxId);
 void Audio_PlayActorSound2(Actor* actor, s16 sfxId);
 // void func_800B8FC0(void);
+void func_800B8F98(Actor* actor, s16 sfxId);
 void func_800B9010(Actor* actor, s16 sfxId);
```

Comment:
```suggestion
void func_800B8E58(Actor* actor, u16 sfxId);
void Audio_PlayActorSound2(Actor* actor, u16 sfxId);
// void func_800B8FC0(void);
void func_800B8F98(Actor* actor, u16 sfxId);
void func_800B9010(Actor* actor, u16 sfxId);
```
I realized while working on another file that sfxIds are suppose to be u16